contract/src/lib.rs

```diff
@@ -340,6 +392,7 @@ impl Contract {
     /// @param merkleProof  merkle tree path (concatenated LE sha256 hashes) (does not contain initial transaction_hash and merkle_root)
     /// @param confirmations how many confirmed blocks we want to have before the transaction is valid
     /// @return True if txid is at the claimed position in the block at the given blockheight, False otherwise
+    #[pause]
```

Comment:
Let's add except
#[pause(except(roles(Role::UnrestrictedVerifyTransaction)))]

Reply:
Added a role

Reply:
Why the role is `UnrestrictedSubmitBlocks` instead of `UnrestrictedVerifyTransaction`?

Reply:
@karim-en Attention issue. Fixed in https://github.com/aurora-is-near/btc-light-client-contract/pull/8/commits/7bf503f0a537fc46a5a441cfaccc5ad6123f17d0